src/vehicle.cpp

```diff
@@ -3567,7 +4022,7 @@ void vehicle::refresh_insides()
         }
         /* If there's no roof, or there is a roof but it's broken, it's outside.
          * (Use short-circuiting && so broken frames don't screw this up) */
-        if( !( part_with_feature( p, "ROOF" ) >= 0 && !parts[ p ].is_broken() ) ) {
+        if( !( part_with_feature( p, "ROOF" ) >= 0 && !parts[ p ].is_available() ) ) {
```

Comment:
! before `is_available` should be removed

Reply:
Yes. Fixed.